repository_service_tuf_worker/repository.py

```diff
@@ -323,8 +323,8 @@ def _get_path_succinct_role(self, target_path: str) -> str:
     def _update_task(
         self,
         bin_targets: Dict[str, List[targets_models.RSTUFTargets]],
-        subtask: AsyncResult,
         update_state: Task.update_state,
+        subtask: Optional[AsyncResult] = None,
```

Comment:
I had to replace where the `subtask` is because I give it a default value and arguments with default values cannot be before arguments without a default value.